addons/website_slides_survey/views/survey_survey_views.xml

```diff
@@ -25,4 +25,21 @@
             </p>
         </field>
     </record>
+
+    <record id='survey_survey_view_form' model="ir.ui.view">
```

Comment:
We usually use double quotes for xml attributes.

Reply:
Fixed it